jcl/src/java.base/share/classes/jdk/internal/foreign/abi/DowncallLinker.java

```diff
@@ -1,6 +1,6 @@
-/*[INCLUDE-IF JAVA_SPEC_VERSION >= 19]*/
+/*[INCLUDE-IF JAVA_SPEC_VERSION == 19]*/
```

Comment:
Is there an issue to track restoring this for jdk20+?

Reply:
As mentioned previously, we already have the issue at https://github.com/eclipse-openj9/openj9/issues/16329 for JEP3434/JDK20 to update all FFI related code in JDK20+.